package.json

```diff
@@ -11,19 +11,26 @@
     "@onaio/connected-private-route": "^0.0.11",
     "@onaio/connected-reducer-registry": "^0.0.3",
     "@onaio/gatekeeper": "^0.1.0",
+    "@onaio/list-view": "^0.0.3",
     "@onaio/loaders": "^0.0.1",
     "@onaio/redux-reducer-registry": "^0.0.9",
     "@onaio/session-reducer": "^0.0.11",
     "@onaio/utils": "^0.0.1",
     "@opensrp/server-logout": "^0.0.1",
     "@opensrp/server-service": "^0.0.4",
+    "@opensrp/user-management": "file:/home/kip/src/ona/web/packages/keycloak-user-management",
```

Comment:
Can we look at this path again?

Reply:
yes its supposed to be "@opensrp/user-management": "file:packages/keycloak-user-management",